src/cpr-mask.component.js

```diff
@@ -53,115 +53,123 @@ export default class InputControl extends React.Component {
 				maskPattern: mask ? mask.pattern : null,
 			});
 		}
-  }
-  render() {
-    return (
-      <div className={`
-        ${this.props.className}
-        ${!this.props.validateMethod(this.state.value) && this.state.isDirty
-            ? this.props.invalidClass
-            : ""}`}>
-        <div style={{display: "flex", alignItems: "center"}}>
-          {this.props.sideChars.left
-            ? <span>{this.props.sideChars.left}</span>
-            : null}
-          <input
-						type="text"
-            ref={(inputRef) => this.input = inputRef}
-            value={this.props.masks.length ? this.state.maskValue : this.state.value}
-            onChange={this.handleChange.bind(this)}
-            onBlur={this.handleBlur.bind(this)}
-            onMouseUp={this.selectionHandle.bind(this)}
-            className={this.props.inputClass}
-            style={{textAlign: this.props.alignment}}
-            onFocus={this.onFocus.bind(this)}
-            placeholder={getPlaceholder(this.props.placeholder, this.props.masks, this.props.validChars, this.props.filler)} />
-          {this.props.sideChars.right
-            ? <span style={{marginLeft: 5}}>{this.props.sideChars.right}</span>
-            : null}
-        </div>
-        {!this.props.validateMethod(this.state.value) && this.state.isDirty
-          ? <span className={this.props.nonValidMsgClass}>{`${this.props.nonValidMsg}`}</span>
-          : null}
-      </div>
-    )
-  }
+	}
+	render() {
+		const inputProps = this.props.inputProps || {};
+		return (
+			<div className={`
+				${this.props.className}
+				${!this.props.validateMethod(this.state.value) && this.state.isDirty
+					? this.props.invalidClass
+					: ""}`}>
+					<div style={{display: "flex", alignItems: "center"}}>
+						{this.props.sideChars.left
+							? <span>{this.props.sideChars.left}</span>
+							: null
+						}
+						<input
+							type="text"
+							ref={(inputRef) => this.input = inputRef}
+							value={this.props.masks.length ? this.state.maskValue : this.state.value}
+							onChange={this.handleChange.bind(this)}
+							onBlur={this.handleBlur.bind(this)}
+							onMouseUp={this.selectionHandle.bind(this)}
+							className={this.props.inputClass}
+							style={{textAlign: this.props.alignment}}
+							onFocus={this.onFocus.bind(this)}
+							placeholder={getPlaceholder(this.props.placeholder, this.props.masks, this.props.validChars, this.props.filler)}
+							{...inputProps}
```

Comment:
This is new. It allows you to override whatever props you want, including things like `autoComplete`, `htmlFor`, etc.